Dockerfile

```diff
@@ -1,7 +1,12 @@
-FROM golang:1.4
+FROM golang:1.7
 
 ADD . /app/src/github.com/Shopify/toxiproxy
-RUN cd /app/src/github.com/Shopify/toxiproxy && GOPATH=/app/src/github.com/Shopify/toxiproxy/Godeps/_workspace:/app go build -ldflags="-X github.com/Shopify/toxiproxy.Version $(cat VERSION)" -o /app/toxiproxy ./cmd
+
+ENV GOPATH /app:$GOPATH
```

Comment:
You're right, it's not using Godeps workspace, but there is a vendor directory already, so Godeps might not be very useful.
It should even be possible to completely remove Godeps from toxiproxy! :)